cargo-workspaces/src/utils/git.rs

```diff
@@ -29,7 +29,7 @@ pub struct GitOpt {
     #[clap(long, conflicts_with_all = &[
         "allow-branch", "amend", "message", "no-git-tag",
         "tag-prefix", "individual-tag-prefix", "no-individual-tags",
-        "no-git-push", "git-remote"
+        "no-git-push", "git-remote", "no-git-commit-msg-branding"
```

Comment:
?

Reply:
Haha, yes, you were too quick for me; I saw this too :)